src/Site.js

```diff
@@ -84,6 +90,9 @@ const SITE_CONFIG_DEFAULT = {
   },
 };
 
+const ABOUT_MARKDOWN_DEFAULT = '# About\n'
+  + 'Welcome to your **About Us** page.\n';
```

Comment:
Just curious, is an "About" page a default page provided by GitHub wikis?

Reply:
Nope, it isn't provided by default. I had actually discussed this with prof @damithc and we decided to include an about.md page as this conversion feature works for both wikis and `docs` folders.

Reply:
I see, thanks for the clarification